[2022-12-28 20:12:41,912] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parking_pipeline.load_data manual__2022-12-28T20:12:36.067345+00:00 [queued]>
[2022-12-28 20:12:41,918] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parking_pipeline.load_data manual__2022-12-28T20:12:36.067345+00:00 [queued]>
[2022-12-28 20:12:41,919] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-28 20:12:41,919] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-28 20:12:41,920] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-28 20:12:41,928] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_data> on 2022-12-28 20:12:36.067345+00:00
[2022-12-28 20:12:41,933] {standard_task_runner.py:52} INFO - Started process 542 to run task
[2022-12-28 20:12:41,937] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'parking_pipeline', 'load_data', 'manual__2022-12-28T20:12:36.067345+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/parking_dag.py', '--cfg-path', '/tmp/tmp8ugkqmyh', '--error-file', '/tmp/tmpb5t6kmo7']
[2022-12-28 20:12:41,938] {standard_task_runner.py:80} INFO - Job 20: Subtask load_data
[2022-12-28 20:12:41,986] {task_command.py:369} INFO - Running <TaskInstance: parking_pipeline.load_data manual__2022-12-28T20:12:36.067345+00:00 [running]> on host 8be693a2d03b
[2022-12-28 20:12:42,044] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=parking_pipeline
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-28T20:12:36.067345+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-28T20:12:36.067345+00:00
[2022-12-28 20:12:42,054] {postgres.py:127} INFO - Running copy expert: COPY parking_data FROM stdin WITH DELIMITER as ',', filename: ./tmp/processed_parking_data_00.csv
[2022-12-28 20:12:42,064] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-12-28 20:12:42,071] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/parking_dag.py", line 89, in _load_parking_data
    filename=filename
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 135, in copy_expert
    cur.copy_expert(sql, file)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type boolean: "activeStatusInd"
CONTEXT:  COPY parking_data, line 1, column activestatusind: "activeStatusInd"

[2022-12-28 20:12:42,082] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=parking_pipeline, task_id=load_data, execution_date=20221228T201236, start_date=20221228T201241, end_date=20221228T201242
[2022-12-28 20:12:42,089] {standard_task_runner.py:97} ERROR - Failed to execute job 20 for task load_data (invalid input syntax for type boolean: "activeStatusInd"
CONTEXT:  COPY parking_data, line 1, column activestatusind: "activeStatusInd"
; 542)
[2022-12-28 20:12:42,109] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-12-28 20:12:42,138] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
